packages/components/src/Menu/Menu.tsx

```diff
@@ -208,15 +214,30 @@ export interface ActionProps {
    * Callback when an action gets clicked
    */
   onClick?(event: React.MouseEvent<HTMLButtonElement>): void;
+
+  /**
+   * Focuses explicitly if set to true
+   */
+  shouldFocus?: boolean;
 }
 
-function Action({ label, icon, onClick }: ActionProps) {
+function Action({ label, icon, onClick, shouldFocus = false }: ActionProps) {
+  // eslint-disable-next-line no-null/no-null
```

Comment:
This is added because a `ref` is not allowed `undefined`. This, we need to specify it some value and the best I could think of was a `null`.

Please do suggest if something else can be used here. Thanks!